prime-dotnet-webapi/ViewModels/Emails/PaperEnrolleeSubmissionEmailViewModel.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace Prime.ViewModels.Emails
+{
+    public class PaperEnrolleeSubmissionEmailViewModel
+    {
+        public string GPID { get; set; }
+
+        public PaperEnrolleeSubmissionEmailViewModel(string gpid) => GPID = gpid;
```

Comment:
you do not need a constructor

Reply:
This was because I am instantiating this VM in the email email service.

`var email = await _emailRenderingService.RenderPaperEnrolleeSubmissionEmail(enrolleeDto.Email, new PaperEnrolleeSubmissionEmailViewModel(enrolleeDto.GPID));`

Changed to the standard constructor